docs/user-guide/ze-profiles.md

```diff
@@ -29,6 +29,25 @@ If you need to edit a profile, click the **Update Profile** button next to the c
 
 In addition, you can hide a profile from the explorer tree, and permanently delete a profile. When you delete your profile permanently, the extension erases the profile from the `.zowe` folder. To hide or delete a profile, right-click the profile and choose one of the respective options from the list.
 
+### Associate Profile
+
+Zowe Explorer allows you to broaden its functionality, using Zowe Explorer Extender APIs. Extenders add new functionality to the extension, leveraging a simple mechanism that enables you to associate profiles. Every Zowe Explorer profile has two types of association: primary and secondary. While the primary association is `zosmf`, the secondary association includes Zowe CLI plug-ins or other services you might have. Within the scope of their own extender package, associated profiles can be used to access a relevant REST API that is available to the extender.
```

Comment:
I think these first two lines are far too general and off the topic at hand. 

Perhaps it would be better to say something along the lines of "Extensions built on Zowe Explorer provide users with additional functionality through unique profile types, such as RSE or FTP. The "associate profiles" function allows users to link, or associate, these extension profile types with an existing zOSMF profile. Every Zowe Explorer profile has two types of association..." Then alter the following sentences to follow this idea...